core-web/libs/sdk/client/src/lib/client/page/page-api.ts

```diff
@@ -365,7 +385,8 @@ export class PageClient {
             return {
                 page: pageResponse,
                 content: contentResponse,
-                errors
+                query: completeQuery,
+                variables: requestVariables
```

Comment:
I think, you can return the params to synchronize both API and GraphQL, UVE handle the rest

Reply:
We can discuss this, I would love to know your opinion on this

Reply:
I'm not sure if we can risk that they add some field that would mess up with the params and it's easier for us and the developer to extract them from a dedicated object

Reply:
I think it's safe to add the params to the page API, we discussed this and it can be breaking change to change the format of the response.